.gitignore

```diff
@@ -23,6 +23,7 @@ package-lock.json
 yarn-error.log
 .DS_Store
 .env
+*.code-workspace
```

Comment:
Why do we need this?

Reply:
Only needed if you happen to store a VS Code workspace in the directory, which I tend to do  regularly.  Could be private, but I don't expect it will be that uncommon.